client/scripts/bundle.js

```diff
@@ -103,14 +101,6 @@ async function main() {
     path.join(ANTLR_MODULE_PATH, SOLIDITY_LEXER_TOKENS),
     path.join(serverAntlrDir, SOLIDITY_LEXER_TOKENS)
   );
-  fs.copyFileSync(
-    path.join(ANTLR_MODULE_PATH, SOLIDITY_TOKENS),
-    path.join(clientAntlrDir, SOLIDITY_TOKENS)
-  );
-  fs.copyFileSync(
```

Comment:
Can you talk me through why we no longer need to copy tokens?

Reply:
We are still copying them to the server folder, I'm not sure why we were also copying them to the client folder in the past, so I removed it

Reply:
I mean, if packaging still works and navigation within the app still works, great.